x/schema/keeper/schema.go

```diff
@@ -15,18 +15,18 @@ import (
 	"github.com/google/uuid"
 	shell "github.com/ipfs/go-ipfs-api"
 	"github.com/sonr-io/sonr/x/schema/types"
-	"github.com/spf13/viper"
 )
 
 var (
-	url        = viper.GetString("IPFS_API_READ")
+	url        = "127.0.0.1:5001"
```

Comment:
@ntindle Will this work in production? I think this is for connecting to the local IPFS shell instance.

@joshLong145 any comment?

Reply:
i have to revert this, good call